pom.xml

```diff
@@ -50,8 +50,8 @@
         <ob-auth.version>1.1.11</ob-auth.version>
         <ob-directory.version>1.5.10</ob-directory.version>
         <ob-analytics.version>1.3.10</ob-analytics.version>
-        <ob-aspsp.version>1.5.4</ob-aspsp.version>
-	<ob-extensions.version>1.5.4</ob-extensions.version>
+        <ob-aspsp.version>1.5.5</ob-aspsp.version>
+	    <ob-extensions.version>1.5.5</ob-extensions.version>
```

Comment:
Extra tab here???

Reply:
Fixed, in my editor it seemed that we miss that tab but finally it was that some hidden character added an extra tab.
I have deleted the line and I created it again to delete the extra tab.